packages/plugins/rum/src/validate.ts

```diff
@@ -101,18 +107,21 @@ export const validateSDKOptions = (options: Options): ToReturn<SDKOptionsWithDef
     return toReturn;
 };
 
-export const validatePrivacyOptions = (options: Options): ToReturn<PrivacyOptionsWithDefaults> => {
+export const validatePrivacyOptions = (
+    options: Options,
+    log: Logger,
+): ToReturn<PrivacyOptionsWithDefaults> => {
     const validatedOptions: RumOptions = options[CONFIG_KEY] || {};
     const toReturn: ToReturn<PrivacyOptionsWithDefaults> = {
         errors: [],
     };
 
     if (validatedOptions.privacy) {
         const privacyWithDefault: PrivacyOptionsWithDefaults = {
-            exclude: [/\/node_modules\//, /\.preval\./],
+            exclude: DEFAULT_EXCLUDE,
```

Comment:
Why the variable?
Also, can you add comments on the regexes you use, just being explicit about what's excluded and what's included.
This way, when someone reads your code, they can confirm the regex does what it's supposed to.